mkt/extensions/models.py

```diff
@@ -359,9 +363,13 @@ def is_version_number_higher(cls, version1, version2):
         return v1 > v2
 
     def publish(self):
-        """Publish this extension version to public."""
+        """Publish this extension version to public.
```

Comment:
This sentence is pretty awkward. Some suggestions:

1. Make this extension version public.
2. Change this extension version's status to public.
3. Publicize this extension version.